books/scss/navbar.scss

```diff
@@ -0,0 +1,31 @@
+.book-navbar {
+  @include shadow(0px 4px 15px 2px $black-shadow);
+  background-color: $white;
+  display: flex;
+  height: 70px;
+  justify-content: center;
+  position: fixed;
+  width: 100%;
+
+  .book-navbar-line {
+    background-color: $cerulean;
+    height: 6px;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  .book-navbar-container {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    max-width: 888px;
```

Comment:
I'm seeing this value repeated, maybe you can move this to a variable?

Reply:
Have been improved